project/RootBuild.scala

```diff
@@ -101,8 +101,8 @@ object RootBuild extends Build {
   // We define this explicitly rather than via unmanagedJars, so that make-pom will generate it in pom.xml as well
   // org % package % version
   val com_adatao_unmanaged = Seq(
-    "com.adatao.unmanaged.net.rforge" % "REngine" % "1.7.2.compiled",
-    "com.adatao.unmanaged.net.rforge" % "Rserve" % "1.7.2.compiled"
+    "com.adatao.unmanaged.net.rforge" % "REngine" % "1.8.2.compiled",
+    "com.adatao.unmanaged.net.rforge" % "Rserve" % "1.8.2.compiled"
```

Comment:
Note: please do NOT merge yet, because I haven't figured out how to build a new version of REngine and Rserve and push to com.adatao.unmanaged.net.rforge